### PR TITLE
fearure-khweb-4/not-found-page

### PR DESCRIPTION
**Pull request by Aleksandra05sh**
Added not found page 404 (https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=3370&projectKey=KHWEB&view=detail&selectedIssue=KHWEB-4) 

- assined  not found page to *
- created component NotFoundPage
- added styles for NotFoundPage

image

### CODE REVIEW

File: src/components/NotFoundPage/NotFoundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from '@mui/material';
+import './NotFoundPage.scss';
+
+export const NotFoundPage = () => (
+  <section className="notfound-page">
```
Comment (Aleksandra05sh) [inline src/components/NotFoundPage/NotFoundPage.tsx:6]: will add Container

File: src/components/NotFoundPage/NotFoundPage.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  & &-button {
+    display: inline-block;
+    background-color: $primary-main;
+    color: $contrast-text-color;
+    font-size: 1.5rem;
+    font-family: $default-font;
+    padding: 1rem 6rem;
+    text-decoration: none;
+    text-transform: uppercase;
+    border-radius: 0.5rem;
```
Comment (vovkis13) [inline src/components/NotFoundPage/NotFoundPage.scss:35]: there is a variable with the default border-radius

File: src/components/NotFoundPage/NotFoundPage.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Container, Link } from '@mui/material';
+import './NotFoundPage.scss';
+
+export const NotFoundPage = () => (
+  <Container maxWidth="xl" className="notfound-page">
+    <h1 className="notfound-page-header">Page not found</h1>
+    <p className="notfound-page-404">404</p>
+    <p className="notfound-page-info">Let&apos;s use another try.</p>
+    <Link className="notfound-page-button" href="/">
```
Comment (vovkis13) [inline src/components/NotFoundPage/NotFoundPage.tsx:10]: use enum Routes

Comment (Aleksandra05sh) [reply src/components/NotFoundPage/NotFoundPage.tsx]: thank you, added

File: src/components/NotFoundPage/NotFoundPage.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+import { Container, Link } from '@mui/material';
+import { RoutesPath } from '../Router/routesPath';
+import './NotFoundPage.scss';
+
+export const NotFoundPage = () => (
+  <Container maxWidth="xl" className="notfound-page">
+    <h1 className="notfound-page-header">Page not found</h1>
+    <p className="notfound-page-404">404</p>
+    <p className="notfound-page-info">Let&apos;s use another try.</p>
+    <Link className="notfound-page-button" href={RoutesPath.HOME}>
```
Comment (kerrimov) [inline src/components/NotFoundPage/NotFoundPage.tsx:11]: not and found are 2 different words

Comment (Aleksandra05sh) [reply src/components/NotFoundPage/NotFoundPage.tsx:11]: are you writing about classname?

Comment (kerrimov) [reply src/components/NotFoundPage/NotFoundPage.tsx:11]: Yes

**Review by kerrimov — APPROVED** [review]

**Pull request merged**